turbinia/config/__init__.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Basic Turbinia config."""
+
+import imp
+import itertools
+import os
+import sys
+
+# Look for config files with these names
+CONFIGFILES = ['.turbiniarc', 'turbinia_config.py']
+# Look in homedir and in the current config dir for config files
+CONFIGPATH = [os.path.expanduser('~'),
+              os.path.dirname(os.path.abspath(__file__))]
+# Config vars that we expect to exist in the configuration
+CONFIGVARS = [
+    'PROJECT',
+    'ZONE',
+    'INSTANCE',
+    'DEVICE_NAME',
+    'SCRATCH_PATH',
+    'BUCKET_NAME',
+]
+# Environment variable to look for path data in
+ENVCONFIGVAR = 'TURBINIA_CONFIG_PATH'
+
+
+class TurbiniaConfigException(Exception):
+  pass
+
+
+def LoadConfig():
+  """Finds Turbinia config file and loads it."""
+  if os.environ.has_key(ENVCONFIGVAR):
+    CONFIGPATH.extend(os.environ[ENVCONFIGVAR].split(':'))
+
+  config_file = None
+  # Load first file found
+  for dir_, file_ in itertools.product(CONFIGPATH, CONFIGFILES):
```

Comment:
So, actually, I just noticed that pylint is complaining about these as invalid variable names now.  Is there a precedent for an underscore prefix on a non-class member?  I was using the underscore in this case more as a way to avoid name collisions with python keywords rather than explicitly as an "internal" variable which I think the prefix underscore is usually used for.   PEP8 also briefly mentions using trailing underscore like this (see single_trailing_underscore): https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles

I could always just use totally different names, but I think there is some precedent for using a trailing underscore in this case.  Let me know what you think.